src/main/scala/battery/bot/core/CommandProcess.scala

```diff
@@ -58,6 +59,59 @@ class CommandProcess(persistenceService: PersistenceService, telegramClient: Tel
           .void
     }
   }
+
+  def sendSettings(sleepingTime: String, wakeUpTime: String, nightCharge: Boolean, result: Update): IO[Unit] = {
+    if (nightCharge) {
+      for {
+        _ <- telegramClient
+          .sendMessage(
+            result.message.chat.id,
+            s"El usuario se levanta a las $sleepingTime, se acuesta a las $wakeUpTime y permite cargar durante la noche"
+          )
+      } yield ()
+    } else {
+      for {
+        _ <- telegramClient
+          .sendMessage(
+            result.message.chat.id,
+            s"El usuario se levanta a las $sleepingTime, se acuesta a las $wakeUpTime y no permite cargar durante la noche"
+          )
+      } yield ()
+    }
+  }
+
+  def checkSettings(result: Update): IO[Unit] = {
+    for {
+      settingsList <- persistenceService.getUserSetting(result.message.from.username)
+      _            <- sendSettings(settingsList._1, settingsList._2, settingsList._3, result)
+    } yield ()
+  }
+
+  def updateSettings(result: Update): IO[Unit] = {
+    val data = result.message.text.split(";").toList
+    data match {
+      case comand :: sleepingTime :: wakeUpTime :: nightCharge :: Nil =>
+        for {
+          _      <- persistenceService.updateUserSleepingTime(result.message.from.username, sleepingTime.toInt)
+          _      <- persistenceService.updateUserWakeUpTime(result.message.from.username, wakeUpTime.toInt)
+          _      <- persistenceService.updateUserNightCharge(result.message.from.username, nightCharge.toBoolean)
```

Comment:
Shouldn't we just update all settings in the same query?

Reply:
Queries joined